Gemfile

```diff
@@ -69,6 +66,11 @@ end
 group :development do
   # Use console on exceptions pages [https://github.com/rails/web-console]
   gem "web-console"
+  gem 'pry-byebug'
+  gem 'awesome_print'
+  gem 'rubocop', require: false
+
+  gem "cssbundling-rails", "~> 1.1"
```

Comment:
This is wrong. The gem will not only be used in production. Tip: Be more careful about groups in the Gemfile.